app/case/server.py

```diff
@@ -64,3 +64,16 @@ def update_status(deed_id):
             return jsonify(case_status=case_status), status.HTTP_200_OK
         else:
             abort(status.HTTP_400_BAD_REQUEST)
+
+    @blueprint.route('/case/<case_id>/deed', methods=['POST'])
+    def update_case_deed(case_id):
+        case = Case.get(case_id)
+
+        if case is None:
+            abort(status.HTTP_404_NOT_FOUND)
+
+        deed_id = request.data['deed_id']
+        case.deed_id = deed_id
+
+        case.save()
+        return str(status.HTTP_200_OK)
```

Comment:
Are we going to be doing a check on whether it was successful from the other side? If not I would say this is wasteful. In my opinion we shouldn't return anything other than the response code keeping the payload as small as possible. I noticed I did the same on the create in case_api, I think I originally did it to test that it worked and forget to take it out. In hindsight I think that the case_api should be responsible for ensuring that the item was created rather than the application that consumes it. 

https://github.com/LandRegistry/charges-case-api/blob/master/app/case/server.py